libraries/chain/include/graphene/chain/protocol/chain_parameters.hpp

```diff
@@ -63,14 +62,47 @@ namespace graphene { namespace chain {
       uint16_t                accounts_per_fee_scale              = GRAPHENE_DEFAULT_ACCOUNTS_PER_FEE_SCALE; ///< number of accounts between fee scalings
       uint8_t                 account_fee_scale_bitshifts         = GRAPHENE_DEFAULT_ACCOUNT_FEE_SCALE_BITSHIFTS; ///< number of times to left bitshift account registration fee at each scaling
       uint8_t                 max_authority_depth                 = GRAPHENE_MAX_SIG_CHECK_DEPTH;
+
+      struct ext
+      {
+         struct committee_updatable_parameters
```

Comment:
Also need hardfork protection in ``committee_member_update_global_parameters_evaluator``, these extensions must remain unset until hardfork.

Reply:
Also need hardfork protection in `proposal_create_evaluator`.

Reply:
I am not averse to adding hardfork protection in proposal_create_evaluator, but I have a question.

Currently the call will throw, as the committee parameters are not set, and can not be set until after hardfork time. That is a round-about way to handle hardfork protection, but it is less code.

So which is better: explicit hardfork protection with more code, or implicit with less code?

Reply:
less code - less bugs :)

Reply:
@jmjatlanta we need (duplicate) hardfork protection code in `proposal_create_evaluator.do_evaluate` because `committee_member_update_global_parameters_evaluator.do_evaluate` won't be called when there is a `committee_member_update_global_parameters_operation` in a proposal.

Reply:
Done. See https://github.com/bitshares/bitshares-core/pull/1370/commits/e8faf8115455d451c843514cec4137c44de07f75